annotation-web/src/main/java/eu/europeana/annotation/web/service/controller/jsonld/BaseJsonldRest.java

```diff
@@ -108,7 +101,7 @@ protected ResponseEntity<String> storeAnnotation(MotivationTypes motivation, boo
         }
 
 	    // 3-6 create ID and annotation + backend validation
-	    webAnnotation.setAnnotationId(annoId);
+	    webAnnotation.setIdentifier(annoIdentifier);
```

Comment:
id generation can be moved to storeMethod, as all ids are generated upon annotation creation